roofit/roofitcore/src/Initialisation.cxx

```diff
@@ -70,7 +70,7 @@ void loadComputeLibrary() {
 
 #if __GNUC__ > 5 || defined(__clang__)
   if (__builtin_cpu_supports("avx512f")) {
```

Comment:
Here, you technically have to check also the other instruction sets that skylake-avx512 supports. On knights landing, this would crash with "Break *** illegal instruction" if one skylake instruction is not supported by knights landing.